Review.swift

```diff
@@ -0,0 +1,294 @@
+            sourceControlService.getTasks(repositoryName: repository.name, pullRequest: pullRequest, req: req).whenSuccess { tasks in
+                formerTasks = tasks
+                
+                logger.info("Creating new directory...")
+                self.createDirectory(pullRequest: pullRequest)
+                
+                logger.info("Downloading sources...")
+                sourceControlService.downloadSources(pullRequest: pullRequest, req: req) {
+                    
+                    logger.info("Unzipping...")
+                    self.unzipSources(fileName: "sourceFiles", fileExtension: ".zip", pullRequest: pullRequest)
+                    
+                    logger.info("Linting...")
+                    let violations = swiftLint.runLinting(for: shortHash, with: project.rules)
+                    var comments: ([StyleViolation], [(StyleViolation, Segment)])?
+                    
+                    logger.info("Getting the Diff...")
+                    sourceControlService.getDiff(repositoryName: repository.name, pullRequestId: pullRequest.scmId, req: req).whenSuccess { diff in
+                        comments = self.getCommentMetaData(violations: violations, diff: diff, onlyAddedLines: true)
+                        
+                        guard let generalComments = comments?.0 else {
+                            logger.warning("No General Comments were generated!")
+                            return
+                        }
+                        guard let inlineComments = comments?.1 else {
+                            logger.warning("No Inline Comments were generated!")
+                            return
+                        }
+                        
+                        logger.info("General Comments: \(generalComments.count)")
+                        logger.info("Inline Comments: \(inlineComments.count)")
+                        
+                        logger.info("Posting Inline Comments...")
+                        for inlineComment in inlineComments {
+                            let comment = Comment(id: nil,
+                                                  line: inlineComment.0.location.line!,
+                                                  lineType: inlineComment.1.type,
+                                                  ruleDescription: inlineComment.0.ruleName,
+                                                  content: inlineComment.0.reason,
+                                                  path: String(inlineComment.0.location.relativeFile?.split(separator: "/", maxSplits: 1, omittingEmptySubsequences: true)[1] ?? ""),
+                                                  type: CommentType(rawValue: inlineComment.0.severity.rawValue))
+                            
+                            sourceControlService.postComment(repositoryName: repository.name, pullRequest: pullRequest, comment: comment, diff: diff, req: req)
```

Comment:
🚫 Error
 [Line Length] At line 77:
Line should be 150 characters or less: currently 159 characters